### ISSUE TITLE
Requesto to change some directories for a default installation

### ISSUE THREAD

**Issue by richard42** [mupen64plus/mupen64plus-user-issues#207]
Originally reported on Google Code with ID 208

```
Describe your system:
 - Linux distribution: Fedora 10
 - Machine type: 64-bit
 - Mupen64Plus version: 1.5
 - Plugins used: All

Hi, I am trying to make an RPM for your excellent emulator but there are
some obstacles that you guys could fix that could facilitate the building:

- Could you include a DESTDIR option in the Makefile so I don't have to
specify every directory where everything should go when typing "make install"?
- Could you make the default installation of the libs in the lib (or for my
case the lib64 directory) directory instead of PREFIX/local/share/mupen64plus?
- Could you do something about the .desktop file where it doesn't only just
look at /usr/local/share/mupen64plus..... for the icon line? I would like
it also if you installed the icons in the systems default icon folder,
/usr/share/icons. Maybe you could have something at the end of the build to
write one up with the appropriate directory that one could specify.
```

Reported by `uncholowapo` on 2009-02-18 15:12:02

**Comment by richard42** [comment]
```
Sorry the default icon destinations is not /usr/share/icons but /usr/share/pixmaps.
```

Reported by `uncholowapo` on 2009-02-18 15:14:55

**Comment by richard42** [comment]
```
All of these requests have been implemented in the new Mupen64Plus 2.0 code.  The
makefiles now include a DESTDIR option, the plugin libs by default go into
/usr/local/lib/mupen64plus, and the desktop file has been removed.  I would love to
see a Fedora package of the upcoming 2.0 release.  The code is almost done; you can
join us at #mupen64plus on irc.freenode.net to discuss packaging issues.
```

Reported by `richard42g` on 2010-01-12 18:00:04
- **Status changed**: `Fixed`
